wordpress-mobile/WordPress-Android
# Update My Site bottom sheet layout

Fixes #11326

This PR updates the layout of the bottom sheet that appears when the FAB from the **My Site** tab is tapped to match the layout of the bottom sheet that appears when the Subfilter bar from the **Reader** tab is tapped:

<img src="https://user-images.githubusercontent.com/830056/74760384-12d90180-5259-11ea-918e-d444fa2158fc.gif" width=300>

## To test

1. Go to **My Site**
2. Tap the FAB
3. Notice the bottom sheet
    - Verify that it has rounded corners, a handle at the top and a title
    - Verify that its layout matches the one from the **Reader**

## Notes

- I haven't felt the need to hide this functionality behind the iA flag, however, you still need to enable the flag if you want to compare the new layout to the one from the **Reader**.
- The possibility to extract a common component that could be used by all bottom sheet implementations will be addressed in a separated issue.

---

PR submission checklist:
